src/email/utils/sesTemplateEmail.ts

```diff
@@ -72,7 +72,7 @@ export const sendBulkEmails = async (data) => {
         } || {},
       ),
     })),
-    // ConfigurationSetName: 'sonu-config',
+     ConfigurationSetName: 'emailRules',
```

Comment:
Yes, This is hardcoded for now, but we need to get it dynamically. we need to discuss if we can get it from our frontend or any other way